src/typechecker/FStar.TypeChecker.Rel.fs

```diff
@@ -963,6 +986,8 @@ let gamma_until (g:gamma) (bs:binders) =
  *   maximal prefix of G_s and G_t, creating a new uvar maximal_prefix(G_s, G_t) |- ?u : t_t,
  *   and assigning ?u_t = ?u
  *
+ * NS: 03/22 Question:  How do we know that t_t is well-formed in maximal_prefix(G_s, G_t)?
```

Comment:
This was a question about the existing code: Probably we should add a check here to make sure that `t_t`'s free variables are in the maximal_prefix